tests/test_pigeon.py

```diff
@@ -435,7 +449,8 @@ async def test_upload(
                     "article_doi": "",
                     "osf_registry": "OSF Registries",
                     "osf_registration_schema": "Open-Ended Registration",
-                    "source": "http://localhost:5000/g752b",
+                    "source": "http://localhost:5000/8gqkv/",
```

Comment:
These values changed while the tests were offline, so they are adjusted now that they are back.